.travis.yml

```diff
@@ -8,23 +8,34 @@ env:
       VERSION: 7
     - DISTRIBUTION: ubuntu
       VERSION: 16.04
+#    - DISTRIBUTION: debian
```

Comment:
Either it's supported and you add those lines uncommented or it's not and there's no need for them.

Reply:
a day, maybe
At least, travis files are equals